pkg/apis/app/v1alpha1/kogitoapp_types.go

```diff
@@ -277,6 +277,8 @@ type Builds struct {
 	// Builds have been stopped from executing
 	// +listType=set
 	Cancelled []string `json:"cancelled,omitempty"`
+	// Flag to let operator know if the given build if from file, set by CLI when user create a kogito service from a Kogito resource, e.g. dmn/drl/bpmn standalone files
+	BuildFromAsset bool `json:"buildFromAsset,omitempty"`
```

Comment:
shouldn't we have an array here ?
New builds could be later defined with Git source or we would have possibility to override a current CR with another source file ?

Reply:
I don't think an array is necessary, all we need is just a bool flag to sinalize to operator that this build is from standalone kogito assets.

Reply:
I don't know if it is possible to update a source file, probably the best option wouldbe remove the service and create it again.

Reply:
@sutaakar wdyt ? should we have this field here or at the root of `Status` part ?

Reply:
I think it is ok to keep it here as it is related to builds.